Sofee.Core.Api/Services/Foundations/Languages/LanguageService.Validations.cs

```diff
@@ -32,6 +32,28 @@ private void ValidateLanguageOnAdd(Language language)
                 Parameter: nameof(Language.UpdatedDate)));
         }
 
+        private void ValidateLanguageOnModify(Language language)
```

Comment:
We need a failing test before this.

Reply:
I messed up the ordering, will do it properly in future PRs.